data/human/human missions.txt

```diff
@@ -4965,3 +4965,185 @@ mission "Riley"
+			choice
+				`	"But what do you want me to do?"`
+				`	"Sorry, but I'd be stupid to accept a mission I don't know anything about."`
+			`	Sophie looks at her brother amusedly. "Of course, my brother just can't wait to get started," She turns back to you. "Maybe I should start with who our father was. We do not know much about him, he came home only occasionally. Don't get me wrong, our parents weren't fighting, but his passion just did not allow him to stay in one place for very long. He was a captain, not unlike you, and he was restless to explore even the furthest parts of the galaxy. We believe he traveled further than most human captains; sometimes he brought small items back home that nobody could explain."`
+			`	She pauses for a moment, lost in her memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. He said that he would not make it easy, though; he was a great fan of fantasy books, and in those stories there is always something mysterious at play. Instead, he left behind a few systems that before we should visit before we go to Prime, where he left something behind for us.`
+			`	"Of course, it would be much easier if we could go directly to Prime, but we want to respect our father's last will." She looks directly at you now, "And that is why we need your help. Could you help us again and take us through all these systems?"`
+			choice
+				`	"Yes, I'd be glad to."`
+						goto accepted
+				`	"I'm sorry, but that sounds too complicated to me."`
+			`	"Well, it looks like we'll need to find somebody else," Sophie says. They both thank you for bringing them this far and walk away.`
+				decline
+			label accepted
+			`	Liam pulls a starmap out of his bag. "These are the systems our father wants us to visit. Apparently they all have something unique to them. We weren't expecting this trip so we'll need to buy some supplies, but as soon as you're ready we'll be on board."`
+				accept
+	on enter "Terminus"
+		dialog `It's often thought that there are nothing to these uninhabited stars by the Deep, but Sophie points out that there's a strange anomaly in this system.`
+	on enter "Aspidiske"
+		dialog `While Sophie and Liam are both quite chatty at times, the beauty of this star stops all of their conversations.`
+	on enter "Gorvi"
+		dialog `Sophie and Liam both talk about how much power a supernova must have, even an old one, if it leaves this much behind.`
+	on enter "Enif"
+		dialog `If there was ever a prime example of how much the human race still has to learn, it is the strange and impressive yet still unfinished Quarg ringworld in this system. Liam tells you, "Our father was a very intriguing man, and he always talked about the Quarg and their ringworlds and how it would be another thousand years before humanity could build anything similar. We never knew how big it was until now!"`
```

Comment:
```suggestion
		dialog `If there was ever a prime example of how much the human race still has to learn, it is the strange and impressive yet still unfinished Quarg ringworld in this system. Liam tells you, "Our father was a very curious man, and he always talked about the Quarg and their ringworlds and how it would be another thousand years before humanity could build anything similar. We never knew how big it was until now!"`
```